builder/builder-config-processor/src/main/java/io/helidon/builder/config/processor/Versions.java

```diff
@@ -1,43 +0,0 @@
-/*
- * Copyright (c) 2023 Oracle and/or its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.helidon.builder.config.processor;
-
-/**
- * Keeps track of the Helidon {@link io.helidon.builder.config.processor.ConfigBeanBuilderCreator} Builder Interop Versions.
- * <p>
- * Since ConfigBean Builder performs code-generation, each previously generated artifact version may need to be discoverable in
- * order to determine interoperability with previous release versions. This class will only track version changes for anything
- * that might affect interoperability - it will not be rev'ed for general code enhancements and fixes.
- * <p>
- * Please note that this version is completely independent of the Helidon version and other features and modules within Helidon.
- */
-public class Versions {
```

Comment:
where is the equivalent of this in the new stuff?  I think this is very important to have from the onset when we break previous version interop, etc.

Reply:
Right now version is hardcoded in `BlueprintProcessor`:
```
            pw.println(GeneratedAnnotationHandler.createString(GENERATOR,
                                                               typeInfo.typeName(),
                                                               prototype,
                                                               "1",
                                                               ""));
```
If we ever decide to change this, we can build it when needed - e.g. the information is there, we just do not need an abstraction for versioning until we start versioning.

Reply:
At least make it a CONST with a note.

Reply:
Follow up created